plugin.video.vrt.nu/resources/lib/kodiwrappers/kodiwrapper.py

```diff
@@ -135,7 +136,12 @@ def get_proxies(self):
         elif httpproxyserver:
             proxy_address = '%s://%s' % (httpproxyscheme, httpproxyserver)
         else:
-            return dict()
+            return None
+
+        if httpproxytype != 0 and HAS_SOCKS is False:
```

Comment:
same here is HAS_SOCKS  a boolean otherwise use not HAS_SOCKS

Reply:
It is a boolean.